### PR TITLE
Unit-test/Travis CI: Cleaner Log, Features; Closes #137

### LINKED ISSUE #137: Unit-test/Travis CI: cleaner log
The current Travis CI log isn't particularly beautiful, [example](https://travis-ci.com/github/jbesomi/texthero/jobs/368704825).

This task consists of cleaning the log output of travis CI by mainly improving the unit-tests files.

Also, sometimes the Travis CI does not fail even if some errors are raised, see example below at line 576 for instance (`raise IOError(Errors.E050.format(name=name))). This should also investigate further. In this case, we may want the job to fail rather than pass?

### PR DESCRIPTION
**Pull request by henrifroese**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,25 +3,35 @@ jobs:
...
+        - pip3 install black
+        - pip3 install --user ".[dev]" .
+
     - name: "Python 3.8.0 on Windows"
       os: windows           # Windows 10.0.17134 N/A Build 17134
       language: shell       # 'language: python' is an error on Travis CI Windows
       before_install:
         - choco install python --version 3.8.0
         - python -m pip install --upgrade pip
+        - pip3 install --no-deps torch===1.6.0 -f https://download.pytorch.org/whl/torch_stable.html
```
Comment (jbesomi) [inline .travis.yml:28]: ?

**Comment by jbesomi** [comment]
Changes in `.travis.yml` are due to a previous PR?

File: texthero/nlp.py [hunk 2]
```diff
@@ -4,8 +4,10 @@
 
 import spacy
 import pandas as pd
+import en_core_web_sm
 
 from texthero._types import TextSeries, InputSeries
+from texthero import stopwords as _stopwords
```
[the remark this answers is not in the document]
Comment (henrifroese) [reply texthero/nlp.py:10]: No, we don't. Will remove

File: texthero/_helper.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def handle_nans(replace_nans_with):
     ... def replace_b_with_c(s):
     ...     return s.str.replace("b", "c")
     >>> s_with_nan = pd.Series(["Test b", np.nan])
-    >>> replace_b_with_c(s_with_nan)
+    >>> replace_b_with_c(s_with_nan) # doctest: +SKIP
```
[the remark this answers is not in the document]
Comment (henrifroese) [reply texthero/_helper.py:46]: Yes, not sure why we're skipping this here.

File: .travis.yml (hunk 1 above)
Comment (henrifroese) [reply .travis.yml:28]: Will remove this (I think we mixed something up in our merge strategy and thought this'll come after flair support (for which we need torch) is implemented

**Comment by henrifroese** [comment]
> Changes in .travis.yml are due to a previous PR?

Yes I think so. Will test some more stuff and incorporate the changes from above soon.

**Comment by henrifroese** [comment]
Now ready to merge from my side :2nd_place_medal:

**Comment by jbesomi** [comment]
Amazing, it looks much more beautiful yet!

There is still a warning: 

> There are NaNs (missing values) in the given input series. They were replaced with appropriate values before the function was applied. Consider using hero.fillna to replace those NaNs yourself or hero.drop_no_content to remove them.

Do you know where it came from? Can we remove it?

Also, I noticed how we don't have a clear position regarding the [CR/CRLF](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) issue (you changed the kind of ending newline in the first two files). I will take care of this by bulk-formating all files and will update the CONTRIBUTION.md accordingly.

**Comment by henrifroese** [comment]
> There is still a warning:

Aaaah, now it all makes sense: that's why we were skipping the doctest in `_helper.py` with the `replace_b_with_c`. That makes sense as a doctest with `warnings.filterwarnings` is really ugly :nauseated_face: and we're testing the function in  `test_helpers.py` anyways. So I'll skip this doctest again and we're good :first_quarter_moon_with_face: 

> Also, I noticed how we don't have a clear position regarding the CR/CRLF issue (you changed the kind of ending newline in the first two files). I will take care of this by bulk-formating all files and will update the CONTRIBUTION.md accordingly.

Yes, makes sense :ok_man:

**Comment by jbesomi** [comment]
Thanks, merged! 🎉  (had to fix some conflicts, but should be good!)

**Pull request merged**